include/nbl/builtin/hlsl/sampling/uniform.hlsl

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2018-2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_BUILTIN_HLSL_SAMPLING_UNIFORM_INCLUDED_
+#define _NBL_BUILTIN_HLSL_SAMPLING_UNIFORM_INCLUDED_
```

Comment:
super misleading header name, there's only spheres here, I'd call the two headers `cos_weighted_spheres` and `uniform_spheres` for clarity

Reply:
ping

Reply:
continued in https://github.com/Devsh-Graphics-Programming/Nabla/pull/899/files?diff=unified&w=1#r2183756414